pdns/backends/gsql/gsqlbackend.hh

```diff
@@ -220,6 +220,7 @@ public:
   bool getComment(Comment& comment);
   void feedComment(const Comment& comment);
   bool replaceComments(const uint32_t domain_id, const string& qname, const QType& qt, const vector<Comment>& comments);
+  string directGSQLQuery(const string &query);
```

Comment:
need to decide on this name - perhaps non-gsql backends also have a use for string commands? (other SQL backends for example)